include/tut/tut_assert.hpp

```diff
@@ -92,7 +96,12 @@ void ensure_not(const M& msg, bool cond)
  * client code will not compile at all!
  */
 template <typename M, typename LHS, typename RHS>
-void ensure_equals(const M& msg, const LHS& actual, const RHS& expected)
+#if __cplusplus >= 201103L
+typename std::enable_if<!std::is_floating_point<LHS>::value && !std::is_floating_point<RHS>::value, void>::type
+#else
+void
+#endif
+    ensure_equals(const M& msg, const LHS& actual, const RHS& expected)
```

Comment:
Maybe not the most beautiful way to do it... same below.
Feel free to comment :)